src/routes/diagrams/index.ts

```diff
@@ -0,0 +1,59 @@
+import { compressAndEncodeBase64, DiagramLanguage, syntaxIsValid } from "./utils";
+import { mermaidDiagramType, mermaidEditorLink, mermaidFormat } from "./mermaid";
+import { plantumlEditorLink } from "./plantuml";
+
+type DiagramDetails = {
+  type: string,
+  editorLink: string,
+  isValid: boolean,
+  imageUrl: string,
+};
```

Comment:
Since we are already fetching SVG to check if its valid can we also add svg here?
Then we can do the #1 in a straight forward way without using the API 2ce.

Reply:
I have resolved issue #1 by replacing the imageURL with diagramSVG. I see the increased speed.